.github/workflows/tests.yaml

```diff
@@ -171,7 +175,28 @@ jobs:
           set -e
           export HELM=${{ steps.helm.outputs.helm-path }}
           make deploy-kyverno-authz-server
+      - name: Install sample app
+        run: |
+          set -e
+          kubectl create ns app
+          kubectl label namespace app istio-injection=enabled
+          kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.24/samples/httpbin/httpbin.yaml -n app
+          kubectl patch deploy httpbin -n app --type=merge -p='{
+            "spec": {
+              "template": {
+                "metadata": {
+                  "labels": {
+                    "ext-authz": "enabled"
+                  }
+                }
+              }
+            }
+          }'
+      - name: Run Chainsaw Tests
+        run:  |
+          set -e
+          chainsaw test tests/e2e/authz-server
       - name: Run Chainsaw Tests
```

Comment:
since these tests are in a different folder, shouldn't this be in a different job?

otherwise, can we rename this step to indicate that it is authz-server test, and can we also rename the test validation-webhook

Reply:
It's not necessary to make it a different job at this point, we can revisit later.